Projekt Python - Education Field.ipynb

```diff
@@ -0,0 +1,340 @@
+       "      <td>Manager</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>4</td>\n",
+       "      <td>5</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>5</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>5</td>\n",
+       "      <td>2</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>10</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>5</td>\n",
+       "      <td>3</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>11</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>5</td>\n",
+       "      <td>4</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>12</th>\n",
+       "      <td>Sales</td>\n",
+       "      <td>Manager</td>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Department  JobRole  JobLevel  Education  EmployeeNumber\n",
+       "0       Sales  Manager         3          1               1\n",
+       "1       Sales  Manager         3          2               1\n",
+       "2       Sales  Manager         3          3               2\n",
+       "3       Sales  Manager         4          1               1\n",
+       "4       Sales  Manager         4          2               3\n",
+       "5       Sales  Manager         4          3               8\n",
+       "6       Sales  Manager         4          4               6\n",
+       "7       Sales  Manager         4          5               2\n",
+       "8       Sales  Manager         5          1               1\n",
+       "9       Sales  Manager         5          2               2\n",
+       "10      Sales  Manager         5          3               4\n",
+       "11      Sales  Manager         5          4               5\n",
+       "12      Sales  Manager         5          5               1"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "newdf = df[(df.Department == 'Sales') & (df.JobRole == 'Manager')].groupby(['Department', 'JobRole', 'JobLevel', 'Education'])['EmployeeNumber'].count().reset_index()\n",
```

Comment:
jeśli ograniczasz do tylko jednego departamentu i jobrole, to po co po nich grupować?